machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -1,11 +1,11 @@
 [id="creating-infrastructure-machinesets"]
```

Comment:
worth dropping an additional `id` in here without the `sets`?

Reply:
We typically try to avoid updating IDs for already released content. However, since the ID is now blatantly wrong, I'll see if I can get that changed.

Reply:
@codyhoag we will need to change the file name too. I think this is an important enough change to go through the process and set up the redirects at the assembly level. Let me know when you are ready. 

Is this going in for 4.5+?

Reply:
Thanks, @vikram-redhat. I'll be sure to go through the process for setting up redirects (and change the file name).

As of now, I believe this is for 4.5+, since there is no clear way for creating infra machines in any supported 4.x version. :+1: